Automattic/jetpack
# Boost: Fix score auto-refresh for cloud CSS

Fixes 48-gh-Automattic/boost-shield-api

#### Changes proposed in this Pull Request:
* Do not auto-refresh score if cloud CSS is still generating
* Auto-refresh speed scores when cloud CSS finishes generating
* Improved detecting when to mark speed scores stale
* Clear transients on boost uninstall.

#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Enable cloud CSS
* Disable and Re-enable cloud CSS
* Make sure speed score doesn't start refreshing while cloud CSS hasn't finished generating
* Wait until cloud CSS finishes generating and see if the speed score auto-refreshes
* Change the theme and come back to Boost settings page. Make sure it fetches fresh speed scores.
